Doc/library/dis.rst

```diff
@@ -1114,8 +1114,13 @@ All of the following opcodes use their arguments.
 .. opcode:: RAISE_VARARGS (argc)
 
    Raises an exception. *argc* indicates the number of arguments to the raise
-   statement, ranging from 0 to 3.  The handler will find the traceback as TOS2,
-   the parameter as TOS1, and the exception as TOS.
+   statement, ranging from 0 to 2. These parameter counts correspond to the 3
+   different forms of the ``raise`` statement:
+
+   * ``0``: ``raise`` (re-raise previous exception).
```

Comment:
We don't need to wrap parameter counts with \`\`\<count\>\`\`.

Reply:
I fix it right now.